### PR TITLE
update links and class/jar name in readme

### PR DESCRIPTION
**Pull request by helenlly**
modify some MD files:
1) some class name missing a "s" . e.g 
the class path should be: com.intel.analytics.zoo.examples.recommendation….
with an extra “s” after “example”

2) some jar name using "zoo-0.1.0-SNAPSHOT-jar-with-dependencies.jar "
,which should be analytics-zoo-VERSION-jar-with-dependencies.jar

3) some python path is updated to "analytics-zoo-VERSION-python-api.zip"

4) some links to Zoo are updated to Analytics Zoo

5) https://github.com/intel-analytics/zoo/issues/293

### CODE REVIEW

File: docs/docs/ScalaUserGuide/run.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ scala> ObjectDetector.loadModel[Float](params.modelPath)
 ## **Run as a Spark Program**
 You can run a analytics zoo program, e.g., the [Object Detection](https://github.com/intel-analytics/analytics-zoo/tree/master/zoo/src/main/scala/com/intel/analytics/zoo/examples/objectdetection), as a standard Spark program (running in either local mode or cluster mode) as follows:
 
-1. Download the pre-trained model from [here](https://github.com/intel-analytics/zoo/tree/master/docs/models/objectdetection).
+1. Download the pre-trained model from [here](https://github.com/intel-analytics/analytics-zoo/tree/master/zoo/src/main/scala/com/intel/analytics/zoo/examples/objectdetection).
```
Comment (hkvision) [inline docs/docs/ScalaUserGuide/run.md]: Why are you linking to this README? This readme has no link to download...

File: pyzoo/zoo/examples/imageclassification/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ This example illustrates how to do the image classification with pre-trained mod
 ### Run steps
 1. Prepare pre-trained models
 
-Download pre-trained models from https://github.com/intel-analytics/zoo/tree/master/docs/models/imageclassification
+Download pre-trained models from [Image Classification](https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/ProgrammingGuide/image-classification.md)
```
Comment (hkvision) [inline pyzoo/zoo/examples/imageclassification/README.md]: Same as object detection.

**Comment by hkvision** [comment]
BTW, have you fixed the link issues raised by intel-analytics/zoo#293 ?

File: docs/docs/ScalaUserGuide/run.md (hunk 1 above)
Comment (helenlly) [reply docs/docs/ScalaUserGuide/run.md]: correct, the right one is https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/ProgrammingGuide/object-detection.md

**Pull request merged**